### PR TITLE
Fixed warnings in jest tests

### LINKED ISSUE #1743: Fix warnings when running unit tests

### PR DESCRIPTION
**Pull request by Calvinwilson99**
- Fixes #1743 

**Description**
Fixed warnings in jest tests.

**Checklist**

- ~~[ ] I have made corresponding changes to the documentation.~~
- ~~[ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added the necessary label (patch/minor/major - If package publish
      is required).~~

### CODE REVIEW

**Comment by Calvinwilson99** [comment]
@ajmaln _a. Please review.
Tippy.js/react does not have the props `onClose` or `distance` so adding them in our components was useless.
I removed `onClose` from `src/components/Dropdown/index.jsx` and `distance` from `src/components/layouts/Sidebar/NavIconWrapper.jsx`.

**Review by ajmaln — COMMENTED** [review]
@Calvinwilson99 _a As discussed over call

- Add a plugin with name `onClose` on tippy.
- Remove the added `div` from Tooltip
- Move the divs to the tests
- Add div around Sidebar `NavIconsWrapper`

**Comment by Calvinwilson99** [comment]
@ajmaln _a. Please check. Made the suggested changes.

File: src/components/layouts/Sidebar/Footer/index.jsx
Comment (ajmaln) [inline src/components/layouts/Sidebar/Footer/index.jsx:1]: @praveen-murali-ind Please check if this would cause any UI issues.

**Review by ajmaln — APPROVED** [review]
@praveen-murali-ind _a Please review

**Review by praveen-murali-ind — APPROVED** [review]
@ajmaln @Calvinwilson99 _a LGTM.

**Pull request merged**